File: trunk/Client/Assets/Script/Panels/PanelMain.cs
using Assets.Script.Frame;
using UnityEngine;
using UnityEngine.UI;

public class PanelMain : PanelBase
{
    public override void Init()
    {
        base.Init();
    }

    public void OnButtonAddCoinClick()
    {
        //Hide();
        PanelManager.me.Get((int) PanelId.PanelGetMoney).Show();
    }

    // Use this for initialization
    private void Start()
    {
    }

    // Update is called once per frame
    private void Update()
    {
    }

    public void OnButtonSettingClick()
    {
        Debug.Log("OnButtonSettingClick");
        //Hide();
        PanelManager.me.Get((int) PanelId.PanelSetting).Show();
    }
}

File: trunk/Tools/LuaEncoder/luajit/PanelMain.lua
local GameObject = UnityEngine.GameObject

PanelMain = {};
local this = PanelMain;

function PanelMain.OnButtonSettingClick()
    -- self.gameObject:SetActive(false)    
    local frame = Assets.Script.Frame
    local panelMgr = frame.PanelManager.me
    local panel = panelMgr:Get(3)
    panel:Show()
    this.gameObject:SetActive(false)
end